ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx

```diff
@@ -67,14 +74,40 @@ export const RuleViolationsStatisticsCard = ({
     return;
   }
 
-  const ruleViolationsTotal = data.pagination.totalCount;
+  const violationsTotal = data.pagination.totalCount;
+
+  const value = jobIncluded
+    ? status === undefined
+      ? '-'
+      : !['FINISHED', 'FINISHED_WITH_ISSUES', 'FAILED'].includes(status)
```

Comment:
See previous.

Reply:
@mmurto Once again, fearless rebasing fails me miserably, so I'll either do a separate fixup commit on top of my PR, or waste another day by rebasing on raw Git.